extension/httpfs/include/s3fs.hpp

```diff
@@ -1,16 +1,15 @@
 #pragma once
 
 #include "httpfs.hpp"
-#include "duckdb/main/database.hpp"
+// TODO: Make this part of duckdb.h, in addition to other httpfs dependencies.
```

Comment:
What does this todo mean exactly?

Reply:
Oh, my point was that fle_opener.hpp has to be included in amalgamation-generated duckdb.h once s3fs becomes real extension. Maybe unnecessary comment. Let me remove it.